docs/manuals/partials/_irods_env.qmd

```diff
@@ -0,0 +1,38 @@
+In order to know with what server iBridges should connect, a so-called iRODS environment file must be present in your home directory.
+
+You may need help from your datamanager or contact Yoda support to obtain the information needed for the config file.
+
+Go to your home directory and create a hidden directory called '.irods':
+
+```
+cd ~
+mkdir .irods
+cd .irods
+```
+
+In this directory, create a file named `irods_environment.json`.
+
+```
+touch irods_environment.json
+```
+Edit the file using a text editor such as nano or vim:
+```
+nano irods_environment.json
+```
+
+Go to the [yoda website](https://www.uu.nl/en/research/yoda/guide-to-yoda/i-am-using-yoda/using-icommands-for-large-datasets), scroll to **Step 2. Configuring iCommands** and copy and paste the text belonging to your institution in the file (similar to the file below) and change the email address next to `irods_user_name`to your yoda user name (typically your uu email address).
```

Comment:
You can also link to the iBridges Tutorial:
https://github.com/UtrechtUniversity/iBridges/blob/develop/tutorials/01-Setup-and-connect.ipynb

The first steps show how to set up the configuration file and also what to check to make really, really sure that it will work for iBridges